doc/source/ray-observability/ray-metrics.rst

```diff
@@ -181,13 +181,13 @@ Ray exports a number of system metrics, which provide introspection into the sta
    * - `ray_node_network_send_speed`
      - `InstanceId`
      - The network send throughput per node, in bytes per second.
-   * - `ray_cluster_active_nodes`
+   * - `ray_autoscaler_active_nodes`
```

Comment:
Btw are there stability guarantees (e.g. `@DeveloperAPI`) for these? Seems weird to advertise them if there aren't guarantees, seems weird to change them if there are guarantees.